tests/files/tests.py

```diff
@@ -340,6 +341,37 @@ def test_file_move_overwrite(self):
         os.close(handle_a)
         os.close(handle_b)
 
+    def test_file_move_copystat_cifs(self):
+        handle_a, self.file_a = tempfile.mkstemp()
+        handle_b, self.file_b = tempfile.mkstemp()
+
+        with mock.patch('django.core.files.move.copystat') as mock_copystat:
```

Comment:
I think it would be better to move `side_effect` to the `mock.patch`, i.e.:
```python
error = PermissionError(errno.EPERM, 'message')
with mock.patch('django.core.files.move.copystat', side_effect=error):
```